inbox/mailsync/backends/gmail.py

```diff
@@ -325,7 +325,14 @@ def resync_uids_impl(self):
                 .options(
                     load_only("msg_uid"), joinedload("message").load_only("g_msgid")
                 )
-                .filter_by(account_id=self.account_id, folder_id=self.folder_id)
+                .filter_by(
+                    ImapUid.account_id == self.account_id,
+                    ImapUid.folder_id == self.folder_id,
+                )
+                .with_hint(
+                    ImapUid,
+                    "FORCE INDEX (ix_imapuid_account_id_folder_id_msg_uid_desc)",
+                )
```

Comment:
Here this is a covering index, since we filter by account_id, folder_id and retrieve msg_uid.